.github/workflows/build-and-test.yml

```diff
@@ -96,7 +99,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --all-features --release
+          args: --all-features
```

Comment:
We sure about this one? Running our tests on debug bug is notoriously slow, I'd say the compilation time difference is way smaller than the execution time difference so not sure if it's worth it?

Reply:
That's surprising. I can revert it, but I think it's unusual that debug vs release would have a meaningful impact on an application like this.

Reply:
I think it's down to hashes, they are very slow on debug mode for some reason

Reply:
For PoW there is a big difference, but since the minimum pow score is 0 in our private tangle it should still be fine?

Reply:
Idk, maybe I just have some PTSDs from bee but I remember debug was just way too slow